controllers/openstackcluster_controller.go

```diff
@@ -353,11 +357,60 @@ func (r *OpenStackClusterReconciler) reconcileNetworkComponents(log logr.Logger,
 }
 
 func (r *OpenStackClusterReconciler) SetupWithManager(mgr ctrl.Manager, options controller.Options) error {
-	return ctrl.NewControllerManagedBy(mgr).
+	controller, err := ctrl.NewControllerManagedBy(mgr).
 		WithOptions(options).
 		For(&infrav1.OpenStackCluster{}).
```

Comment:
Hm what do you think, should we just use the same code as CAPA (with more log statements :))? (also in the machine controller)